spec/units/emergency_banner_spec.rb

```diff
@@ -1,22 +1,25 @@
 RSpec.describe GovukWebBanners::EmergencyBanner do
-  let(:redis_client) { double(hgetall: {}) }
-  subject(:emergency_banner) { GovukWebBanners::EmergencyBanner.new(redis_client:) }
+  subject(:emergency_banner) { GovukWebBanners::EmergencyBanner.new }
+
+  before do
+    Rails.application.config.emergency_banner_redis_client = double(hgetall: {})
```

Comment:
I think another way to do this would be to make the `redis_client` method in the class public and stub that. `allow .... to receive` etc.

I'm sure we can play around with nicer ways to write this.

But I do prefer this commit.

Reply:
I think probably (previous comments notwithstanding) we shouldn't tinker with the model too much just to make the tests nicer. If you think this is understandable, I'm okay with it.

Reply:
I think we can play around with the stubbing to make that nicer.